changelog.d/7691.bugfix

```diff
@@ -0,0 +1 @@
+Fix an incorrect type in the `GET room_keys/version` endpoint.
```

Comment:
We've been trying to include info about when a bug was introduced in the changelogs -- or has this just always been this way?

Reply:
It's pretty much always been that way (ever since that field was included in the result)

Reply:
```suggestion
Fix a long standing bug where the response to `GET room_keys/version` endpoint had the incorrect type for the `etag` field.
```